### DISCUSSION TITLE
Which is best way of programming on your country?

### DISCUSSION THREAD

**Discussion by iosonoagenda** [codewars/codewars.com#2979] [Q&A]
Every country has its own methods of programming, it's like a style that can be useful for professionals, like me, to make the difference between each others. What's your method? Tell all your secrets! Or most of them ;)

**Comment by iosonoagenda** [comment]
Yeah, it's so damn true. I am an expert IT with 16 years of experience. I have so much to learn and so much time to do that. Every single day there is a new challenge and I am here to help others find best path to solutions in best times! People says I am a genius, I say I am a simple kind man who only wants the best for everybody. My best is even to help others, if I can help myself, I can help others, not?

**Reply by samoodi** [reply]
That's so kind. Keep it up.
